inc/Config/DataSource/HttpDataSource.php

```diff
@@ -37,6 +37,10 @@ final public function get_service_name(): string {
 		return static::SERVICE_NAME;
 	}
 
+	final public function get_config_schema_version(): int {
```

Comment:
Doesn't seem like this is needed, I don't see it called anywhere

Reply:
I haven't incorporated it in, that's why. Once I finish off this PR, I'll see if its still needed